x-pack/libbeat/autodiscover/providers/aws/elb/watch.go

```diff
@@ -66,11 +65,12 @@ func (w *watcher) forever() {
 // once executes the watch loop a single time.
 // This is mostly useful for testing.
 func (w *watcher) once() error {
-	ctx, _ := context.WithTimeout(context.Background(), w.interval)
+	ctx, _ := context.WithTimeout(context.Background(), w.period)
 	fetchedLbls, err := w.fetcher.fetch(ctx)
 	if err != nil {
 		return err
 	}
+	logp.Info("fetched %d load balancer listeners from AWS for autodiscovery", len(fetchedLbls))
```

Comment:
Maybe logging this on every loop is too much, log this at the debug level?

Reply:
Btw, in any case I think this should be `Infof` so it formats the string.
```suggestion
	logp.Infof("fetched %d load balancer listeners from AWS for autodiscovery", len(fetchedLbls))
```

Reply:
Thanks, will convert it to debug thanks!